### DISCUSSION TITLE
[v4] ally bug

### DISCUSSION THREAD

**Discussion by rodrigogaraujo** [adonisjs/core#3007] [Help]
I try login with google 
https://vccuida.g3infotech.app/api/login/google
"@adonisjs/ally": "2.1.1",
"@adonisjs/ally": "2.1.3",
"@adonisjs/ally": "2.2.0",

But always return this message:
OAuthException: E_OAUTH_STATE_MISMATCH: Oauth state mis-match

Can somebody help me?

**Comment by thetutlage** [comment]
Set `sameSite = false` inside the cookies config

**Reply by rodrigogaraujo** [reply]
Its true, 
is there a way to use social login + jwt login?
